### ISSUE TITLE
How is y predicted from the values of the leaf in a XG boost regression in python

### ISSUE THREAD

**Issue by MugdhaKiran** [bonehead/Higgs-Boson#1]
I am relatively new to XG boost regression and need help understanding how the xg boost model predicts the value of y on a test data set from the value of the leaves in python.

I have built a XGB Regression model. I have provided the code and the leaf I am seeing below. 
![download](https://user-images.githubusercontent.com/53262797/61793002-aec59a80-ae3b-11e9-93cf-8bd72397e020.png)

The mean value of my y_train is 0.81. However, the values that I am seeing in the leaves is too small. I have already set the base score as 0.0

`import numpy as np
import pandas as pd
import csv as csv
from matplotlib import pyplot
from sklearn.model_selection import GridSearchCV
from sklearn.model_selection import train_test_split
from sklearn.model_selection import cross_val_score,KFold
from sklearn.metrics import mean_absolute_error
import matplotlib.pyplot as plt 
from scipy.stats import skew
from collections import OrderedDict
import xgboost
from xgboost import plot_importance

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, random_state=123)

best_xgb_model = xgboost.XGBRegressor(colsample_bytree=0.4,base_score=0.0, gamma=0,  learning_rate=0.07, max_depth=3, min_child_weight=10, n_estimators=1000,                                                                reg_alpha=0.75, reg_lambda=0.45, subsample=0.95, seed=42)

y_pred = best_xgb_model.predict(X_test)

from xgboost import plot_tree
plot_tree(best_xgb_model, num_trees= 4, rankdir= 'LR')
plt.rcParams['figure.figsize'] = [60, 60]
plt.show()`
